doc-content/shared-kie-docs/src/main/asciidoc/KieServer/kie-server-commands-samples-ref.adoc

```diff
@@ -0,0 +1,876 @@
+      "container" : {
+        "status" : null,
+        "messages" : [ ],
+        "container-id" : "command-script-container",
+        "release-id" : {
+          "version" : "1.0",
+          "group-id" : "com.redhat",
+          "artifact-id" : "Project1"
+        },
+        "config-items" : [ ]
+      }
+    }
+  } ]
+}
+----
+
+.Example server response (JSON)
+[source,json]
+----
+{
+  "response": [
+    {
+      "type": "SUCCESS",
+      "msg": "Container command-script-container successfully deployed with module com.redhat:Project1:1.0.",
+      "result": {
+        "kie-container": {
+          "container-id": "command-script-container",
+          "release-id": {
+            "version" : "1.0",
+            "group-id" : "com.redhat",
+            "artifact-id" : "Project1"
+          },
+          "resolved-release-id": {
+            "version" : "1.0",
+            "group-id" : "com.redhat",
+            "artifact-id" : "Project1"
+          },
+          "status": "STARTED",
+          "scanner": {
+            "status": "DISPOSED",
+            "poll-interval": null
+          },
+          "config-items": [],
+          "messages": [
+            {
+              "severity": "INFO",
+              "timestamp": {
+                "java.util.Date": 1538762455510
+              },
+              "content": [
+                "Container command-script-container successfully created with module com.redhat:Project1:1.0."
+              ]
+            }
+          ],
+          "container-alias": null
+        }
+      }
+    }
+  ]
+}
+----
+--
+GetContainerInfoCommand::
++
+--
+Returns information about a specified KIE container in {KIE_SERVER}.
+
+.Command attributes
+[cols="30%,50%,20%", frame="all", options="header"]
+|===
+|Name
+|Description
+|Requirement
+
+|`containerId`
+|ID of the KIE container
+|Required
+
+|
+|
+|Optional
+|===
+
+.Example Java command
+[source,java]
+----
+
+----
+
+.Example REST request body (JSON)
+[source,json]
+----
+{
+  "commands" : [ {
+    "get-container-info" : {
+      "container-id" : "command-script-container"
+    }
+  } ]
+}
+----
+
+.Example server response (JSON)
+[source,json]
+----
+{
+  "response": [
+    {
+      "type": "SUCCESS",
+      "msg": "Info for container command-script-container",
+      "result": {
+        "kie-container": {
+          "container-id": "command-script-container",
+          "release-id": {
+            "group-id": "com.redhat",
+            "artifact-id": "Project1",
+            "version": "1.0"
+          },
+          "resolved-release-id": {
+            "group-id": "com.redhat",
+            "artifact-id": "Project1",
+            "version": "1.0"
+          },
+          "status": "STARTED",
+          "scanner": {
+            "status": "DISPOSED",
+            "poll-interval": null
+          },
+          "config-items": [
+
+          ],
+          "container-alias": null
+        }
+      }
+    }
+  ]
+}
+----
+--
+ListContainersCommand::
```

Comment:
single attribute kieContainerResourceFilter of type KieContainerResourceFilter that is optional

Reply:
Done.